Sources/MountebankSwift/Models/Recreatable.swift

```diff
@@ -0,0 +1,176 @@
+import Foundation
+
+// MARK: - Protocol
+
+public protocol Recreatable {
+    func swiftString(depth: Int) -> String
+}
+
+extension Recreatable {
+    public var debugSwiftString: String {
+        swiftString(depth: 0)
+    }
+}
+
+let DELIMITER = ",\n"
+let INDENTATION = "    "
+
+// MARK: - Helpers
+
+extension Recreatable {
```

Comment:
These are all helpers to create a kind of formatted, indented string that is the exact string that is needed to create the object in swift.